go/teams/handler.go

```diff
@@ -140,14 +140,20 @@ func handleSBSSingle(ctx context.Context, g *libkb.GlobalContext, teamID keybase
 	case keybase1.TeamInviteCategory_EMAIL:
 		// nothing to verify, need to trust the server
 	case keybase1.TeamInviteCategory_KEYBASE:
-		// XXX and check the seqno
 		uv, err := invite.KeybaseUserVersion()
 		if err != nil {
 			return err
 		}
 		if uv.Uid.NotEqual(invitee.Uid) {
 			return fmt.Errorf("chain keybase invite link uid %s does not match uid %s in team.sbs message", uv.Uid, invitee.Uid)
 		}
+		if !uv.EldestSeqno.Eq(invitee.EldestSeqno) {
+			if uv.EldestSeqno == 0 {
+				g.Log.CDebugf(ctx, "team.sbs invitee eldest seqno: %d, allowing it to take the invite for eldest seqno 0 (reset account)")
```

Comment:
What does this mean? If someone invites `dbb165b7879fe7b1174df73bed0b9500%5` to a team, their client should not allow `dbb165b7879fe7b1174df73bed0b9500%0` into the team.

Reply:
this is the reverse.  xxx%0 was invited, and xxx%6 is accepting the invitation.

Reply:
Ah, ok!